### ISSUE TITLE
Need help with poodle?

### ISSUE THREAD

**Issue by Codeman655** [Codeman655/COSC583#1]
I can't possibly finish this in time. Can one of you get it?

**Comment by Codeman655** [comment]
I definitely won't be able to get to the actual poodle attack by Sun.
I'll probably be able to get padding_oracle attacks and downgrade attacks.
I still need to read up on SSL, but I'm finishing my slides Mon/Tue of next week.

**Comment by mduwalsh** [comment]
I was reading up on SSL and looks like we are not going deep though. We are
leaving messages part off. Are we leaving that part off of project? It is
too much details I guess. I'll finish my report part on ssl1, ssl2, ssl3
and their comparison by Sunday night.
So, I think we should meet Tuesday afternoon to discuss/write on slides.

**Comment by Codeman655** [comment]
I thought we were going to get into the message section. @awachte was currently working on the exchange portion and I hoped your part on how each version worked would explain HTTP message encryption (Since v3 actually has the method of encryption part of the protocol). Yeah, everyone's busy, and I'd like to finish this as fast as possible.

I saw your comments on `history.md.` Is that the comment you were intending to add?

**Comment by mduwalsh** [comment]
Yes, I can finish message section (message authentication code, encryption
and creating cryptographic parameters) by tonight.

Thanks,
Mahendra
